packages/react-reconciler/src/ReactFiberCompleteWork.js

```diff
@@ -1052,9 +1052,6 @@ function completeWork(
             return null;
           } else {
             // This is a Hoistable Instance
-
-            // This must come at the very end of the complete phase.
-            bubbleProperties(workInProgress);
```

Comment:
What's this change? Was this never necessary?

Reply:
shoot, bad edit, wasn't supposed to come out